RxExample/RxExample/Services/GeolocationService.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  GeolocationService.swift
+//  RxExample
+//
+//  Created by Carlos García on 19/01/16.
+//  Copyright © 2016 Krunoslav Zaher. All rights reserved.
+//
+
+import Foundation
+import CoreLocation
+#if !RX_NO_MODULE
+    import RxCocoa
+#endif
+
+class GeolocationService {
+    
+    static let instance = GeolocationService()
+    private (set) var autorized: Driver<Bool>
+    private (set) var location: Driver<CLLocationCoordinate2D>
+    
+    private let locationManager = CLLocationManager()
+    
+    private init() {
+        locationManager.distanceFilter = kCLDistanceFilterNone;
+        locationManager.desiredAccuracy = kCLLocationAccuracyBestForNavigation;
+        
+        autorized = locationManager.rx_didChangeAuthorizationStatus
+            .startWith(CLLocationManager.authorizationStatus())
```

Comment:
I think this would need to have form
```
Observable.deferred {
    locationManager
       .rx_didChangeAuthorizationStatus
       .startWith(CLLocationManager.authorizationStatus())
}
.asDriver(....
```
... otherwise we are using value at time of declaration, and not subscription.